airflow/providers/google/cloud/operators/kubernetes_engine.py

```diff
@@ -898,3 +898,36 @@ def execute(self, context: Context):
         ).fetch_cluster_info()
 
         return super().execute(context)
+
+
+class GKEStartKueueJobOperator(GKEStartJobOperator):
+    """
+    Executes a Kubernetes Job in Kueue in the specified Google Kubernetes Engine cluster.
+
+    :param queue_name: The name of the Queue in the cluster
+    """
+
+    def __init__(
+        self,
+        *,
+        queue_name: str,
+        **kwargs,
+    ) -> None:
+        super().__init__(**kwargs)
+        self.queue_name = queue_name
+
+        if self.suspend is False:
+            raise AirflowException(
+                "The `suspend` parameter can't be False. If you want to use Kueue for running Job"
+                " in a Kubernetes cluster, set the `suspend` parameter to True.",
+            )
+        elif self.suspend is None:
+            warnings.warn(
+                f"You have not set parameter `suspend` in class {self.__class__.__name__}. "
+                "For running a Job in Kueue the `suspend` parameter should set to True.",
+                AirflowProviderDeprecationWarning,
+                stacklevel=2,
+            )
```

Comment:
when `suspend ` always required as True why don't you just hardcode it in init?
```python
if not self.suspend:
    raise AirflowException(f"suspend param can't be {self.suspend}")
self.suspend = True
```

Reply:
I want to make it more informative for users and also I want to exclude cases when the user uses in the operator `suspend=False` and the operator rewrites this to `True` without any information to the user.

About your code suggestion it does not make sense, because in Airflow we stop importing DAG when raising an Exception inside the `__init__` method.  It means that this code will assign `suspend=True` only in case when suspend is equal to True.

Reply:
Warning makes sense in this case and yes - more explicit messages are good -> @pankajastro WDYT ?

Reply:
Yeah. sometimes less is more, but when it comes to user confusion, often less is just .... less :)